otri/analysis/__init__.py

```diff
@@ -1,9 +1,33 @@
 
 from typing import Any, Sequence
-
+from sqlalchemy.orm.session import Session
+from sqlalchemy.orm.query import Query
 from ..filtering.stream import Stream
 
 
+def db_share_query(session: Session, atoms_table: str, ticker: str, provider: str) -> Query:
```

Comment:
Don't think such method should be in the analysis module, it probably belongs to a future common database interface.

Decide yourself whether you want to keep it for later moving or remove it now.